agent/githubHook.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8
+import github3
+import json
+import os
+from getpass import getpass
+
+KEY_LABEL = 'iskakalunan'
+STATE_SUCCESS = 'success'
+STATE_PENDING = 'pending'
+
+
+class githubHook(object):
+    def __init__(self):
+        user, password = self._get_credentials()
+        g = github3.login(user, password)
+        self.hasal = g.repository('Mozilla-TWQA', 'Hasal')
+
+    def _get_credentials(self):
+        github_credential = 'github.json'
```

Comment:
Issue #682 is introduced for using json format file to decide runner on executing.
Issue #683 is introduced for using authorization token.